policyengine_us/parameters/gov/aca/age_curves/ny.yaml

```diff
@@ -0,0 +1,19 @@
+description: Age curve factors for ACA premium pricing in New York
```

Comment:
```suggestion
description: New York multiples xyz... under the ACA premium pricing.
```

Could we use more descriptive descriptions in the yaml files - 
applies to all parameters here